packages/stark-ui/src/modules/app-logout/components/app-logout.component.ts

```diff
@@ -0,0 +1,75 @@
+import { Component, Inject, Input, OnInit, ViewEncapsulation } from "@angular/core";
+
+import {
+	STARK_LOGGING_SERVICE,
+	STARK_ROUTING_SERVICE,
+	STARK_APP_CONFIG,
+	StarkLoggingService,
+	StarkRoutingService,
+	StarkApplicationConfig,
+	STARK_SESSION_SERVICE,
+	StarkSessionService
+} from "@nationalbankbelgium/stark-core";
+
+/**
+ * Name of the component
+ */
+const componentName: string = "stark-app-logout";
+
+/**
+ * Component to display the application's logout button
+ */
+@Component({
+	selector: "stark-app-logout",
+	templateUrl: "./app-logout.component.html",
+	styleUrls: ["./app-logout.component.scss"],
```

Comment:
We don't use encapsulation in stark-ui. Each components have 2 possible scss importable files by the end user, in this case:
- `_app-logout-theme.scss`: which contains the customization of the Material theme (colors and fonts)
- `_app-logout.component.scss`: which contains all the rest

Then this line can be removed and you can add this to `showcase/src/styles/_theme.scss`:
```@import "~@nationalbankbelgium/stark-ui/src/modules/app-logout/components/app-logout.component";```